### PR TITLE
SUBMARINE-945. Create helm chart for submarine operator

### PR DESCRIPTION
**Pull request by MortalHappiness**
### What is this PR for?
Create helm chart for submarine operator

### What type of PR is it?
[Feature]

### What is the Jira issue?
https://issues.apache.org/jira/projects/SUBMARINE/issues/SUBMARINE-945

### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? No

### CODE REVIEW

File: submarine-cloud-v2/helm-charts/submarine-operator/.helmignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```
Comment (pingsutw) [inline submarine-cloud-v2/helm-charts/submarine-operator/.helmignore:1]: Need to add the license

Comment (MortalHappiness) [reply submarine-cloud-v2/helm-charts/submarine-operator/.helmignore:18]: @pingsutw I have added the license, thank you.

**Comment by kevin85421** [comment]
LGTM @pingsutw Do you have any other recommendations?

**Review by pingsutw — APPROVED** [review]
+1

**Pull request closed without merging**